sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -374,11 +374,6 @@ private RxDocumentClientImpl(URI serviceEndpoint,
         this.diagnosticsClientConfig.withConsistency(consistencyLevel);
         this.throughputControlEnabled = new AtomicBoolean(false);
 
-        logger.info(
-            "Initializing DocumentClient [{}] with"
-                + " serviceEndpoint [{}], connectionPolicy [{}], consistencyLevel [{}], directModeProtocol [{}]",
-            this.clientId, serviceEndpoint, connectionPolicy, consistencyLevel, configs.getProtocol());
-
```

Comment:
Another option I considered was to just add the additional info being logged here and have the start-up time only logged from the CosmosClientBuilder but felt odd to do so. Here one can't calculate the total time yet since there are other calls happening before and after from the CosmosClientBuilder.